Migration.md

```diff
@@ -0,0 +1,351 @@
+# Migrate from Lob’s JavaScript to TypeScript SDK
+
+This guide illustrates differences between Lob’s current JavaScript library (lob-node) and our new TypeScript SDK (lob-typescript-sdk). The new TypeScript SDK supports all generally available endpoints and has the benefit of being built with TypeScript.
+We have also compared performance and found the new TypeScript SDK is equivalent or improved in most cases with some operations experiencing a consistent 2X improvement.
+In this guide we compare the following method pattern.
+
+- CREATE
+- LIST
+- GET
+- DELETE
+- BULK VERIFY (ADDRESS VERIFICATION)
+- VERIFY (BANK ACCOUNTS)
+  ​
+
+## INSTALL
+
+​
+Similar to lob-node, the lob-typescript-sdk package is available through NPM:
+
+```
+$ npm install lob
+```
+
+## IMPORT AND INITIALIZE
+
+Lob-node uses JavaScript’s built-in require. You initialize by passing your API key as an argument:
+
+```javascript
+const Lob = require("lob")("YOUR API KEY");
+​
+```
+
+The new TypeScript SDK uses an ES module import statement and configuration variable for the object containing your API key. This variable is used to instantiate a new configuration:
+
+```typescript
+import {
+  Configuration,
+  AddressesApi,
+  AddressEditable,
+} from "lob-typescript-sdk";
```

Comment:
Let's remove AddressesApi and AddressEditable

These are not needed for the sample below

Reply:
Done!